src/textual/messages.py

```diff
@@ -45,12 +45,24 @@ def can_replace(self, message: Message) -> bool:
 
 @rich.repr.auto
 class Layout(Message, verbose=True):
+    """Sent by Textual when a layout is required."""
+
     def can_replace(self, message: Message) -> bool:
         return isinstance(message, Layout)
 
 
+@rich.repr.auto
+class UpdateScroll(Message, verbose=True):
+    """Sent by Textual when a scroll update is required."""
+
+    def can_replace(self, message: Message) -> bool:
+        return isinstance(message, UpdateScroll)
+
+
 @rich.repr.auto
 class InvokeLater(Message, verbose=True, bubble=False):
+    """Sent by Textual to invoke a callback."""
```

Comment:
Take a look at how `Button.Pressed` is documented, for example, and consider following that format. That will make it a breeze to generate the online documentation for the messages.

https://github.com/Textualize/textual/blob/main/src/textual/widgets/_button.py#L162-L170

Reply:
I'll come back to that. These should really be private. Or at least something to indicated to devs you shouldn't touch them.